src/AdminPanel/modules/LoginPage/LoginFormFields.tsx

```diff
@@ -0,0 +1,32 @@
+import { ErrorMessage } from '@hookform/error-message';
+import { useFormContext } from 'react-hook-form';
+import { LoginFormFieldsNames, LoginFormTypes } from './types';
+
+export const LoginFormFields = () => {
+  const {
+    register,
+    formState: { errors },
+  } = useFormContext<LoginFormTypes>();
+
+  return (
+    <div>
```

Comment:
maybe you could change div to React fragment

Reply:
resolved